common/djangoapps/student/views.py

```diff
@@ -124,6 +124,7 @@
 
 # Note that this lives in openedx, so this dependency should be refactored.
 from openedx.core.djangoapps.credentials.utils import get_user_program_credentials
+from openedx.core.djangoapps.credit.email_utils import get_credit_provider_display_names
```

Comment:
this utility method is not specific to email only as we are using it for dashboard as well so could you move it in https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/credit/utils.py